### PR TITLE
Offcanvas changes

### PR DESCRIPTION
**Pull request by ffsiren**
fixes: DT-435, DT-395

### CODE REVIEW

File: app/component/navigation/index-navigation.scss [hunk 1]
```diff
@@ -326,3 +321,7 @@ ul.offcanvas-list {
     font-weight: $font-weight-bold;
   }
 }
+
+.offcanvas > div {
+  z-index:2000 !important;
```
Comment (samuliheljo) [inline app/component/navigation/index-navigation.scss]: :) ?

File: server/server.js [hunk 2]
```diff
@@ -129,6 +129,7 @@ function setUpRoutes() {
     var locale = req.cookies.lang  || req.acceptsLanguages(['fi', 'sv', 'en']) || 'en';
     var messages = translations[locale]
     var context = application.createContext()
+    navigator = {userAgent:req.headers['user-agent']};  //required by material-ui
```
Comment (samuliheljo) [inline server/server.js]: what is this? where is it used?

File: app/component/navigation/index-navigation.scss [hunk 3]
```diff
@@ -129,10 +129,13 @@ nav.top-bar {
...
 }
 
-.left.off-canvas {
+.left-off-canvas {
+  padding: 0;
+  background: #333;
   padding-top: 20px;
   color: $offcanvas-link-color;
   height:100%;
+  overflow: scroll;
```
Comment (samuliheljo) [inline app/component/navigation/index-navigation.scss]: as discussed...

File: server/server.js (hunk 2 above)
Comment (ffsiren) [inline server/server.js]: Server side rendering of material-ui components needs the global 'navigator' to be set

**Pull request merged**